kafka-ballerina/Ballerina.toml

```diff
@@ -1,42 +1,72 @@
-[project]
-org-name = "ballerinax"
+[package]
+org = "ballerinax"
+name = "kafka"
 version = "@toml.version@"
 
-[dependencies]
-"ballerina/config" = "@stdlib.config.version@"
-"ballerina/file" = "@stdlib.file.version@"
-"ballerina/io" = "@stdlib.io.version@"
-"ballerina/log" = "@stdlib.log.version@"
-"ballerina/runtime" = "@stdlib.runtime.version@"
-"ballerina/stringutils" = "@stdlib.stringutils.version@"
-"ballerina/system" = "@stdlib.system.version@"
-"ballerina/time" = "@stdlib.time.version@"
-
-[platform]
-target = "java11"
-
-    [[platform.libraries]]
-    artifactId = "kafka"
-    version = "@project.version@"
-    path = "../kafka-native/build/libs/kafka-native-@project.version@.jar"
-    groupId = "org.ballerinalang.messaging.kafka"
-    modules = ["kafka"]
-
-    [[platform.libraries]]
-    artifactId = "kafka-clients"
-    version = "@kafka.version@"
-    path = "./lib/kafka-clients-@kafka.version@.jar"
-    groupId = "org.apache.kafka"
-
-    [[platform.libraries]]
-    artifactId = "kafka_2.12"
-    version = "@kafka.version@"
-    path = "./lib/kafka_2.12-@kafka.version@.jar"
-    groupId = "org.apache.kafka"
-
-    [[platform.libraries]]
-    artifactId = "avro"
-    version = "1.9.2"
-    path = "./lib/avro-1.9.2.jar"
-    groupId = "org.apache.avro"
-    scope = "compile"
+[[dependency]]
+org = "ballerina"
+name = "io"
+version = "@stdlib.io.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "config"
+version = "@stdlib.config.version@"
+
+[[dependency]]
+org = "ballerina"
+name = "time"
```

Comment:
We can remove transitive dependencies from here.

Reply:
Fixed in 7250f71184245b3f7cb6e2b311ce3a416901a1c1